jtag_registers.xml

```diff
@@ -43,56 +43,69 @@
         The size of this register will remain constant in future versions so
         that a debugger can always determine the version of the DTM.
         <field name="0" bits="31:17" access="R" reset="0" />
-        <field name="dbusreset" bits="16" access="W1" reset="0">
-            Writing 1 to this bit resets the dbus controller, clearing any
+        <field name="dmireset" bits="16" access="W1" reset="0">
+            Writing 1 to this bit resets the DMI controller, clearing any
             sticky error state.
         </field>
         <field name="0" bits="15" access="R" reset="0" />
         <field name="idle" bits="14:12" access="R" reset="Preset">
-            The number of cycles a debugger needs to send the target through
-            Run-Test/Idle after every dbus scan.
+            This is a hint to the debugger of the minimum number of cycles
```

Comment:
This is not just a hint. It's a minimum. Some implementations may only do something in the Run-Test/Idle state. This makes for a slower but simpler implementation of the TAP. And if there's some state machine there, eg. 1 state for read, and another for write, then you would always have to spend 2 cycles in Run-Test/Idle.

It can also be used, as you described, just to avoid seeing a busy state and that is, as you say, not super important.